### ISSUE TITLE
Replace Bell Emoji with basic Unicode Character

### ISSUE THREAD

**Issue by devyntk** [JustDerek1/FRCCentralIssues#23]
The notification emoji sticks out quite a bit when viewing the app, it seems like you could just replace it simply with this unicode character: http://www.fileformat.info/info/unicode/char/1f514/index.htm

**Comment by brendanarciszewski** [comment]
If you wanted, the same could be done with the heart (but I'm indifferent to the heart):http://m.unicode-table.com/en/sets/hearts-symbols/

**Comment by JustDerek1** [comment]
I am going to look into this as it is a simple enhancement.  I think I might be using the character that you are pointing me to. It may be iOS that is rendering it more vividly. Apple does that with some Unicode characters.

**Comment by JustDerek1** [comment]
Yep, that is the unicode character that I am already using.  If you click on the image, it will take you here:  http://www.fileformat.info/info/unicode/char/1f514/browsertest.htm

On Mac OS and iOS, you will see that it is the same icon that I am currently using.  I'm not saying it is good.  It's just the one that I'm using.
